examples/step-68/parameters.prm

```diff
@@ -9,13 +9,13 @@ subsection Particle Tracking Problem
   set Output directory              = ./
 
   # Iteration frequency at which output results are written
-  set Output frequency              = 10
+  set Output inverse frequency              = 10
```

Comment:
Just call it "Output interval" or "Time steps between output" or some such.

Reply:
Yeah that's right, output interval is fine and dandy :). I'll use that instead.